src/Styling/Home.css

```diff
@@ -29,16 +25,18 @@
 }
 
 #Hero_center_child_3 {
-  margin: 10px 20px 0px 0px;
+  margin: 10px 10px 0px 0px;
   text-align: right;
 }
 
 .Scroll_indicator {
-  margin-bottom: 20px;
-  grid-area: 4 / 2 / span 1 / span 2;
+  margin: 0px 0px -10px 70px;
   display: flex;
   flex-direction: column;
-  justify-content: center;
+  position: absolute;
+  justify-content: end;
+  align-self: end;
+  height: 240px;
```

Comment:
I would consider using rem/em instead of px in your styles. Extends to be less responsive